### PR TITLE
Enable django admin for more apps

### PR DESCRIPTION
**Pull request by 2e2a**

### CODE REVIEW

File: meinberlin/apps/maps/admin.py [hunk 1]
```diff
@@ -1,31 +1,7 @@
...
-                             'To initialize a new areasetting enter the '
-                             'string "false" without quotation marks.')
-        })
-    )
-
-    def module_name(self, setting):
-        return setting.module.name
-
-    def project_name(self, setting):
-        return setting.module.project
```
Comment (xi) [inline meinberlin/apps/maps/admin.py]: Maybe it would be better to leave this here until it is merged into core.

**Comment by xi** [comment]
LGTM.

File: meinberlin/apps/maps/admin.py (hunk 1 above)
Comment (kleingeist) [reply meinberlin/apps/maps/admin.py]: if you are already preparing a PR for core i think it is okay to delete it with this PR in meinBerlin

**Comment by kleingeist** [comment]
lgtm :)
thanks for the work

**Pull request merged**